.travis.yml

```diff
@@ -144,7 +144,7 @@ script:
     - /tmp/julia/bin/julia -e 'versioninfo()'
     - pushd /tmp/julia/share/julia/test
     - /tmp/julia/bin/julia --check-bounds=yes runtests.jl $TESTSTORUN &&
-      /tmp/julia/bin/julia --check-bounds=yes runtests.jl libgit2-online download pkg
+      /tmp/julia/bin/julia --check-bounds=yes runtests.jl libgit2-online download
```

Comment:
`.circleci/config.yml` need to be changed as well?

Reply:
Yes, I didn't know that existed, thanks.